performant-software/digital-fauvel
# Render annotation text and connect to toggle menu

### What does the PR do?
Upon Mirador's completion of annotation-loading for a page, the plugin will now render text attached to the annotation on top of its rectangle along with a semi-opaque backdrop for visibility. The text layer menu control now opens into a dropdown allowing the user to switch among no annotations and rendered text in three transcription modes (English, French, original). 

Also adds French and original transcription links to the manifest file.

### What issues does it address?
Closes #9, closes #10, closes #14, closes #39 

### How to test
Open a page in Image View for which there are transcriptions. You should see the hit boxes for the transcription line annotation objects appear briefly, then be replaced by overlaid text. By clicking on the text layer dropdown in the upper right, you should be able to toggle between different languages or turn the text/annotation display off entirely.
